airbyte-integrations/connectors/destination-mongodb-strict-encrypt/src/test-integration/resources/mongodb-cert.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
+MIIDOTCCAiECFGAxiWb4qfyQIgo4eGN2xML8CBapMA0GCSqGSIb3DQEBCwUAMFkx
+CzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRl
+cm5ldCBXaWRnaXRzIFB0eSBMdGQxEjAQBgNVBAMMCTEyNy4wLjAuMTAeFw0yMjA3
+MjkxMDEwMDRaFw0yMzEyMTExMDEwMDRaMFkxCzAJBgNVBAYTAkFVMRMwEQYDVQQI
+DApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQx
+EjAQBgNVBAMMCTEyNy4wLjAuMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoC
+ggEBALFqVajw2b6vPWsIcEh3+SxBsf4zEbm6f0L83AS1RH3+zSLQFcd5Bk886Uk5
+F98b2q6bfEKE1KT9PaUiRukq7carLmEyVqlGwWTHFoT756R3626u3zFgaLrqi+ss
+YVS3cy5BgACOdo27G5VjERwukj+9UKZcyBhobzZ0BQgiosMrHOeR2zja6HKf+qju
+9q6DmKKkVjqM1fbV9WK45tKzAZ3Quy1JXCT1BfLhkVXK3B0LsuX2JnmPVp7O5cAF
+aj/7hyAMmvI9ZnZWIB5LleiSTLI3QTAyv+j1rYZtPo7M4gceGRKLHIDKu8wL+kEt
+j8sZJTxifKeFSjPwBYwbgZNnHSUCAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAFzDl
+pxrcHJFUgf4rNXFvND+iq4K1AtqEnd4Ze7N3pSHEIUj4YNdiMZP06nP5FAJklRkP
+GsMMeR8s+jfZtPgTlwrW2IKcdYZAkA91VoEDs6IK2VzC7GiEsUPaGh4/2/DjXkSZ
+9Ie4a4zfRVHCvkRMO5q9/KKpdTo1vwc83mmxMSKQKEv8NKbEhVc3VlCZV1MvpMdc
+7He0Jqfg53+yLZv33w6NA4X8QIWdoBuK02ZP7ZhEBcMJOByetsuaHXjadsOpP4LM
+7KaP6zUq9kteKD6TgiCTmV6U/tpDvhS3YdVBEdd0aPlBeWGJ+GzubbOUUJOYSkt0
+jQ/Z56qy7eSveCa5ZQ==
+-----END CERTIFICATE-----
```

Comment:
What is the expiration date of these keys? We need them to be valid for a very long time.